data/validation-tagged.yaml

```diff
@@ -0,0 +1,37 @@
+# COCO 2017 dataset http://cocodataset.org
```

Comment:
What is this file for? An example?

Reply:
yes, the locally used example, I removed it from git